cdap-ui/app/features/tracker/tracker.less

```diff
@@ -272,3 +280,246 @@ body.state-tracker {
   }
 
 }
+
+body.state-tracker-entity {
+  .tracker-entity-top-bar {
+    display: table;
+    height: 50px;
+    width: 100%;
+    border-bottom: 1px solid @border-color;
+    background-color: white;
+
+    .back-button, .entity-detail, .tab-options, .action-button {
+      height: 100%;
+      vertical-align: middle;
+      display: table-cell;
+    }
+
+    .back-button {
+      width: 50px;
+      border-right: 1px solid @border-color;
+      cursor: pointer;
+    }
+
+    .entity-detail {
+      padding-left: 10px;
+      width: 200px;
+      h3 {
+        font-size: 16px;
+        margin: 0;
+      }
+      span {
+        color: @tracker-grey;
+        font-size: 12px;
+      }
+      .entity-type {
+        margin-top: 3px;
+      }
+    }
+
+    .tab-options {
+      ul {
+        margin-bottom: 0;
+        height: 50px;
+      }
+      li {
+        font-size: 14px;
+        font-weight: 500;
+        margin-right: 40px;
+        padding: 0;
+        height: 100%;
+        line-height: 50px;
+
+        &.active {
+          a { color: @title-font-color; }
+          border-bottom: 4px solid @tracker-green;
+        }
+        a {
+          color: @tracker-grey;
+          text-decoration: none;
+          vertical-align: middle;
+        }
+      }
+    }
+
+    .action-button {
+      padding-right: 15px;
+    }
+  }
+
+  main.container {
+    h4 {
+      font-size: 14px;
+      color: @title-font-color;
+    }
+
+    .no-properties-message {
+      margin-top: 150px;
+      h4 { color: @tracker-grey; }
+    }
+  }
+
+  .entity-container {
+    padding: 20px;
+  }
+
+  .entity-detail-tags-column, .entity-detail-schema-column, .entity-detail-properties-column {
+    position: absolute;
+    top: 178px;
+    bottom: 53px;
+    box-sizing: border-box;
```

Comment:
small but nitpicky change: use the box-sizing mixin to ensure all vendor prefixes are accounted for - .box-sizing(border-box);